### PR TITLE
feat: add alaz to emerging applications

### PR DESCRIPTION
**Pull request by fatihbaltaci**
Adding [Alaz](https://github.com/ddosify/alaz) to eBPF emerging applications. The old PR is closed because of merge conflicts, @xmulligan.

Old PR: https://github.com/ebpf-io/ebpf.io-website/pull/512

### CODE REVIEW

**Comment by xmulligan** [comment]
Everything looks fine except it looks like you are linking to the company website rather than the project website. Does the project have a website?

**Comment by fatihbaltaci** [comment]
@xmulligan Alaz only has a GitHub page, it does not have a website. Do you want me to remove the website field?

**Comment by xmulligan** [comment]
Yes, please remove the website then

File: src/data/pages/applications/emerging.js [hunk 1]
```diff
@@ -283,6 +283,16 @@ const emergingApplications = [
...
     ],
   },
+  {
+    logoUrl: 'https://github.com/ddosify/alaz',
+    name: 'Alaz',
+    logoName: 'alazLogo',
+    title: 'Effortless, Low-Overhead, eBPF-based Kubernetes Monitoring',
+    description: `Alaz is an open-source Ddosify eBPF agent that can inspect and collect Kubernetes (K8s) service traffic 
+    without the need for code instrumentation, sidecars, or service restarts.
+    Alaz uses eBPF to create a Service Map that helps identify golden signals and problems like high latencies, 5xx errors, zombie services, slow HTTP requests and SQL queries.`,
```
Comment (xmulligan) [inline src/data/pages/applications/emerging.js:293]:
```suggestion
    description: `Alaz is an open source Ddosify eBPF agent that can inspect and collect Kubernetes service traffic 
    without the need for code instrumentation, sidecars, or service restarts.
    Alaz uses eBPF to create a Service Map that helps identify golden signals and problems like high latencies, 5xx errors, zombie services, slow HTTP requests, and SQL queries.`,
```

**Comment by xmulligan** [comment]
Sorry lost this in the lead to KubeCon. Last few grammar nits

**Review by xmulligan — APPROVED** [review]

**Comment by xmulligan** [comment]
Thanks!

**Pull request merged**